requirements.txt

```diff
@@ -1,41 +1,40 @@
 Babel==2.3.4
-decorator==4.0.10
-docutils==0.12
+decorator==4.0.11
+docutils==0.13.1
 ebaysdk==2.1.5
 feedparser==5.2.1
-gevent==1.1.2 ; sys_platform != 'win32'
-greenlet==0.4.10
+gevent==1.1.2
+greenlet==0.4.11
 html2text==2016.9.19
 Jinja2==2.8
 libsass==0.12.3
 lxml==3.7.1 ; sys_platform != 'win32'
 lxml ; sys_platform == 'win32'
-Mako==1.0.4
+Mako==1.0.6
 MarkupSafe==0.23
 mock==2.0.0
-num2words==0.5.4
+num2words==0.5.6
 ofxparse==0.16
-passlib==1.6.5
+passlib==1.7.0
 Pillow==4.0.0
-psutil==4.3.1; sys_platform != 'win32'
-psycopg2==2.7.3.1; sys_platform != 'win32'
+psutil==5.0.1
+psycopg2==2.7.3.1
 pydot==1.2.3
 pyldap==2.4.28; sys_platform != 'win32'
 pyparsing==2.1.10
 PyPDF2==1.26.0
-pyserial==3.1.1
+pyserial==3.2.1
 python-dateutil==2.5.3
 pytz==2016.7
 pyusb==1.0.0
-PyYAML==3.12
 qrcode==5.3
 reportlab==3.3.0
-requests==2.11.1
-suds-jurko==0.6
+requests==2.12.4
+suds-jurko==0.7
```

Comment:
on a clean install, windows 10, python 3.6 32 bits, `suds-jurko` 0.7 is not found:

> Could not find a version that satisfies the requirement suds-jurko==0.7 (from -r requirements.txt (line 33)) (from versions: 0.4.1.jurko.1, 0.4.1.jurko.2, 0.4.1.jurko.3, 0.4.1.jurko.4, 0.4.1.jurko.5, 0.5, 0.6)
No matching distribution found for suds-jurko==0.7 (from -r requirements.txt (line 33))